XenModel/InvisibleMessages.zh-CN.resx

```diff
@@ -133,7 +133,7 @@
     <value>https://cis.citrix.com/legal/</value>
   </data>
   <data name="HOMEPAGE" xml:space="preserve">
-    <value>https://www.citrix.com/products/citrix-hypervisor</value>
+    <value>https://www.citrix.com.cn/products/citrix-hypervisor/</value>

```

Comment:
I think the Japanese resx needs also correction to https://www.citrix.co.jp/products/citrix-hypervisor/

Reply:
I will look into this. The '.cn' link exists but there is no '.jp' or '.ja' link which exists.

Reply:
Found the correct link for japenese. Changes should be coming now.

Reply:
The link I pasted above does exist - note it's `.co.jp` not `.com.jp`.

Reply:
O yeah, Thank you!